### DISCUSSION TITLE
raspberry pi pico flashing

### DISCUSSION THREAD

**Discussion by sagittarius0815** [ttan/Mute-o-Matic-V2#2] [Q&A] [answered]
As i understand you flashed the pico with "circuitpython"
When i try to run the code.py I've got an error 
```
line 6
no module with name 'adafruit_hid'
```

Could you explain the steps how to flash the pico?

**Comment by ttan** [comment, marked as answer]
Hi, you need to install adafruits libraries to run - I didn’t write that explicitly but thanks for the reminder, I’ll update the read me. 
the full budndle is available [here](https://github.com/adafruit/Adafruit_CircuitPython_Bundle/releases/tag/20210217), extract it and copy the adafruit_hid file in a ‘lib’ folder on the Raspberry

**Reply by sagittarius0815** [reply]
Thanks, now its working!
